pre-docs/writing_tasks.md

```diff
@@ -359,7 +361,9 @@ Successful on 1 node: localhost
 Ran on 1 node in 0.12 seconds
 ```
 
-### Ruby Example
+#### Ruby Example
```

Comment:
This is nested under "Task Helpers", but the example is just a generic multi-file task, it doesn't actually show using the `ruby_task_helper`.

Reply:
The example was already there, as a way of showing how to send other helper files. I thought it'd be good to have one example using a task helper (the Python one) and another example showing another ruby library. I'm not sure how to title the sections differently to indicate that?

Reply:
Maybe we should have an example with the ruby task helper and separately have an example of how to generically use extra files? I think both of those things are useful to document.